pylib/cqlshlib/test/test_cqlsh_output.py

```diff
@@ -653,6 +653,7 @@ def test_describe_columnfamily_output(self):
                 varintcol varint
             ) WITH additional_write_policy = '99p'
                 AND allow_auto_snapshot = true
+                AND auto_snapshot_ttl = 0s
```

Comment:
default is 0s which means no TTL. There can not be negative values and 0 TTL does not make sense which makes 0s good candidate.